dataset/dictionary/characters.json5

```
@@ -525,8 +525,8 @@
     ja: "バーべロス",
     zhCN: "芭比洛斯",
     tags: [ "mondstadt", "character-sub" ],
-    notes: "魔女会・コードB。バーベロスがモナの師匠 (おばば) という説もあるが明言されておらず、おばばはバーベロスの後継者という説もある。",
-    notesZh: "魔女会代号B。虽然已经明说芭比洛斯是莫娜的师傅（老太婆），但是也有老太婆是芭比洛斯的继承人的说法。",
+    notes: "魔女会・コードB。モナの師匠 (おばば)", // TODO: v4.8 期間限定イベントのクエストムービーによりモナの師匠であることが確定
+    notesZh: "魔女会代号B。莫娜的师傅（老太婆）", // TODO: v4.8 限时事件任务电影确认自己是莫娜的导师
```

```suggestion
    notes: "魔女会・コードB。モナの師匠 (おばば)", // In a quest movie in the limited-time event of v4.8 in Simulanka, it is explicitly mentioned that Barbelos and Mona's old hag are identical person.
    notesZh: "魔女会代号B。莫娜的师傅（老太婆）", // v4.8 限时事件任务电影确认自己是莫娜的导师
```

- Let's remove TODO sign because no further action is required for these lines.
- For comments, English comment is preferred to Japanese so that non-Japanese speakers (usually Chinese speakers) can understand.